examples/02_meta-analyses/10_plot_cbmr.py

```diff
@@ -0,0 +1,362 @@
+"""
+import numpy as np
+import scipy
+from nilearn.plotting import plot_stat_map
+
+from nimare.generate import create_coordinate_dataset
+from nimare.meta import models
+from nimare.transforms import StandardizeField
+
+###############################################################################
+# Load Dataset
+# -----------------------------------------------------------------------------
+
+# data simulation
+ground_truth_foci, dset = create_coordinate_dataset(foci=10, sample_size=(20, 40), n_studies=1000)
+# set up group columns: diagnosis & drug_status
+n_rows = dset.annotations.shape[0]
+dset.annotations["diagnosis"] = [
+    "schizophrenia" if i % 2 == 0 else "depression" for i in range(n_rows)
+]
+dset.annotations["drug_status"] = ["Yes" if i % 2 == 0 else "No" for i in range(n_rows)]
+dset.annotations["drug_status"] = (
+    dset.annotations["drug_status"].sample(frac=1).reset_index(drop=True)
+)  # random shuffle drug_status column
+# set up continuous moderators: sample sizes & avg_age
+dset.annotations["sample_sizes"] = [dset.metadata.sample_sizes[i][0] for i in range(n_rows)]
+dset.annotations["avg_age"] = np.arange(n_rows)
+# set up categorical moderators: schizophrenia_subtype (as not enough data to be interpreted
+# as groups)
+dset.annotations["schizophrenia_subtype"] = ["type1", "type2", "type3", "type4", "type5"] * int(
+    n_rows / 5
+)
+dset.annotations["schizophrenia_subtype"] = (
+    dset.annotations["schizophrenia_subtype"].sample(frac=1).reset_index(drop=True)
+)  # random shuffle drug_status column
+
+###############################################################################
+# Estimation of group-specific spatial intensity functions
+# -----------------------------------------------------------------------------
+# Unlike kernel-based CBMR methods (e.g. ALE, MKDA and SDM), CBMR provides a
+# generative regression model that estimates a smooth intensity function and
+# can have study-level moderators. It's developed with a spatial model to
+# induce a smooth response and model the entire image jointly, and fitted with
+# different variants of statistical distributions (Poisson, Negative Binomial
+# (NB) or Clustered NB model) to find the most accurate but parsimonious model.
+#
+# CBMR framework can generate estimation of group-specific spatial internsity
+# functions for multiple groups simultaneously, with different group-specific
+# spatial regression coefficients.
+#
+# CBMR framework can also consider the effects of study-level moderators
+# (e.g. sample size, year of publication) by estimating regression coefficients
+# of moderators (shared by all groups). Note that moderators can only have global
+# effects instead of localized effects within CBMR framework. In the scenario
+# that there're multiple subgroups within a group, while one or more of them don't
+# have enough number of studies to be inferred as a separate group, CBMR can
+# interpret them as categorical study-level moderators.
```

Comment:
I might specifically state what the upcoming model is. That is, what are the specific moderators, group categories, etc...

Reply:
Thanks for the suggestions! I've rephrased to make it more specific